src/ynViewer.sol

```diff
@@ -1,20 +1,23 @@
 // SPDX-License-Identifier: BSD 3-Clause License
 pragma solidity ^0.8.24;
 
-import {IynETH} from "src/interfaces/IynETH.sol";
-import {IynViewer} from "src/interfaces/IynViewer.sol";
-import {IStakingNodesManager,IStakingNode} from "src/interfaces/IStakingNodesManager.sol";
+import {IDelegationManager, IStrategy} from "@eigenlayer/src/contracts/interfaces/IDelegationManager.sol";
+import {IEigenPod} from "@eigenlayer/src/contracts/interfaces/IEigenPod.sol";
+import {IERC4626} from "@openzeppelin/contracts/interfaces/IERC4626.sol";
+
+import {IStakingNode, IStakingNodesManager, IynViewer} from "./interfaces/IynViewer.sol";
 
 contract ynViewer is IynViewer {
-    IynETH public ynETH;
-    IStakingNodesManager public stakingNodesManager;
+
+    IERC4626 public immutable ynETH;
```

Comment:
ynETH is not a IERC4626 for now and may never be one, it doesn't break any of the logic here, but i'd rather not confuse.

Let's leave it with IynETH

Reply:
https://github.com/yieldnest/yieldnest-protocol/pull/119/commits/216eef7e6babcaf745b59f4a8516fc535a175b47